src/main/scala/nl.knaw.dans.easy.bagstore/server/StoresServletComponent.scala

```diff
@@ -161,9 +161,9 @@ trait StoresServletComponent extends DebugEnhancedLogging {
             )))
             .getOrRecover {
               case e: IllegalArgumentException => BadRequest(e.getMessage)
-              case e: BagIdAlreadyAssignedException => BadRequest(e.getMessage)
               case e: NoBagException => BadRequest(e.getMessage)
               case e: InvalidBagException => BadRequest(e.getMessage)
+              case e: BagIdAlreadyAssignedException => BadRequest(e.getMessage)
```

Comment:
is there a specific reason for moving this exception down 2 lines? Probably this is a change left over from previous attempts?

Reply:
You are right, this can be reverted.